index.html

```diff
@@ -10,13 +10,13 @@
     <div id="wrapper">
       <header>
         <div id="logo">
-          <p id="logoPlaceHolderText">@ YOUR HOTEL</p>
+          <img id="logo" src="Pictures/checkInLogo.svg" />
          </div>
 
         <ul id="headerNav">
           <li><a href="index.html">HOME</a></li>
           <li><a href="roomselection.html">SELECT ROOM</a></li>
-          <li>CONTACT</li>
+          <span id="hidden"><li>CONTACT</li></span>
```

Comment:
`hidden` should be a class here, applied directly to the `li` instead of on a surrounding `span`. It's not legal for `ul` to directly contain a `span`.